### DISCUSSION TITLE
Progetto esame Open Source - Luca Evangelisti

### DISCUSSION THREAD

**Discussion by lucaeva03** [deneb-alpha/Lab_open_source-2023_2024#30] [Projects]
**Proposta progetto**

Parte principale:
1.  Modelli di Business nel mondo Open Source: spiegazione delle varie tipologie accompagnata da esempi
2. Localizzazione del Wiki con il Team italiano di traduzione

Parte facoltativa:
1. Riproduzione dei bug con Bugzilla

La ringrazio e le auguro una buona giornata.

**Comment by deneb-alpha** [comment]
Buongiorno Luca,
ci vediamo tra poco, in bocca al lupo!
